jupyterhub_singleuser_profiles/ui/src/SizesForm/SizesForm.tsx

```diff
@@ -96,30 +91,33 @@ const SizesForm: React.FC<ImageFormProps> = ({ uiConfig }) => {
       ['Default'],
     );
 
-    return sizes.reduce((acc, size) => {
-      const sizeDescription = sizeDescriptions?.find((desc) => desc?.name === size);
-      if (!sizeDescription) {
-        acc.push(
-          <SelectOption
-            key={size}
-            value={size}
-            description="Resources set based on administrator configurations"
-          />,
-        );
-      } else if (sizeDescription.schedulable !== false) {
-        acc.push(
-          <SelectOption
-            key={size}
-            value={size}
-            description={
-              `Limits: ${sizeDescription.resources.limits.cpu} CPU, ${sizeDescription.resources.limits.memory} Memory ` +
-              `Requests: ${sizeDescription.resources.requests.cpu} CPU, ${sizeDescription.resources.requests.memory} Memory`
-            }
-          />,
-        );
-      }
-      return acc;
-    }, [] as React.ReactElement[]);
+    const defaultSelection = (
+      <SelectOption
+        key="Default"
+        value="Default"
+        description="Resources set based on administrator configurations"
+      />
+    );
+
+    return sizes.reduce(
+      (acc, size) => {
+        const sizeDescription = sizeDescriptions?.find((desc) => desc?.name === size);
+        if (sizeDescription && sizeDescription.schedulable !== false) {
```

Comment:
since `sizeDescriptions` has the option to return `undefined` and be destructured ... it might be good to apply optional chains or break out `sizeDescription` possibly set a default value for `resources`... optional chaining may be quicker `sizeDescription?.resources...` 

or...
```
const { schedule, resources, ...somethingElse } = sizeDescriptions?.find...
```

Reply:
resolved (i don't have perms to close this comment)